cairo/tests/src/utils/test_dict.py

```diff
@@ -1,11 +1,32 @@
+from typing import List, Mapping, Tuple
+
 import pytest
+from ethereum_types.numeric import Uint
+from hypothesis import given
+from hypothesis import strategies as st
+
+from tests.utils.strategies import felt
 
 pytestmark = pytest.mark.python_vm
 
 
-@pytest.mark.parametrize(
-    "test_case",
-    ["test__dict_copy__should_return_copied_dict"],
-)
-def test_dict(cairo_run, test_case):
-    cairo_run(test_case)
+@given(dict_entries=st.lists(st.tuples(felt, felt, felt)))
```

Comment:
```suggestion
@given(dict_entries=...)
```

isn't enough?

Reply:
I specifically need to generate integers in the valid felt range